src/graphql/operations/proposals.ts

```diff
@@ -55,9 +55,19 @@ export default async function (parent, args) {
     params.push(`%${where.plugins_contains}%`);
   }
 
+  if (where.strategies_in) {
+    searchSql += " AND JSON_OVERLAPS(JSON_EXTRACT(p.strategies, '$[*].name'), JSON_ARRAY(?))";
```

Comment:
Refer to https://github.com/snapshot-labs/snapshot-hub/pull/704#discussion_r1371279625